.travis.yml

```diff
@@ -6,18 +6,16 @@ script: "bundle exec rake validate && bundle exec rake lint && bundle exec rake
 matrix:
   fast_finish: true
   include:
-  - rvm: 1.9.3
-    env: PUPPET_GEM_VERSION="~> 3.4.0"
-  - rvm: 1.9.3
+  - rvm: 1.8.7
```

Comment:
I think this module specifically didn't work well with 1.8.7, so we weren't testing with it. But otherwise :+1: